Makefile

```diff
@@ -20,17 +25,51 @@ PKGMAP = $(P_TIMESTAMP),$(P_ANY)
 protos:
 	cd pb/protos && PATH=$(PATH):$(GOPATH)/bin protoc --go_out=$(PKGMAP):.. *.proto
 
+
+##
+## Testing
+##
+
+OPENBAZAARD_NAME ?= openbazaard-$(GIT_TAG)-$(GIT_SHA)
+BITCOIND_PATH ?= .
+
+.PHONY: openbazaard
```

Comment:
This isn't a phony make target.

Reply:
agreed

Reply:
I was concerned that the build output makes an `openbazaard-<GITTAG>-<GITSHA>` file... not `openbazaard`. Does that matter? (Not a makefile expert.)